commands.js

```diff
@@ -2081,15 +2081,15 @@ exports.commands = {
 				if (!stdout) {
 					connection.popup("The modlog is empty. (Weird.)");
 				} else {
-					connection.popup("|wide||html|<p>The last " + lines + " lines of the Moderator Log of " + roomNames + ":</p>" + stdout);
+					connection.popup("|wide||html|<p>The last " + lines + " lines of the Moderator Log of " + roomNames + ".</p><p><small>[" + Tools.toTimeStamp(new Date(), {hour12: true}) + "]</small> Current server time.</p>" + stdout);
```

Comment:
I'm gonna suggest "← current server time"

The capitalization and period are very out of place for a sentence fragment.